addons/l10n_dk/models/account_account.py

```diff
@@ -0,0 +1,20 @@
+from odoo import api, models, _
+from odoo.exceptions import UserError
+
+
+class AccountAccount(models.Model):
+    _inherit = ['account.account']
+    
+    @api.ondelete(at_uninstall=False)
+    def _unlink_bank_cash_accounts(self):
+        if self.env.company.country_code != 'DK':
```

Comment:
Maybe we need to use account_fiscal_country_id ? :thinking:

Reply:
I also thought about doing something like this

```suggestion
        if self.env.company.account_fiscal_country_id != self.env.ref('base.dk'):
```

But I'm not sure if it's the best approach perf wise. Maybe the final reviewer can give us more insight 😁